get-started/index.md

```diff
@@ -115,7 +115,7 @@ in Linux for a long time. Docker has worked to make these capabilities approacha
     a simple container, this talk doesn't go into networking, using images for the filesystem, 
     and more. But, it gives a _fantastic_ deep dive into how things are working.
 
-    <iframe width="560" height="315" src="https://www.youtube-nocookie.com/embed/8fi7uSYlOdc" frameborder="0" allow="accelerometer; autoplay; encrypted-media; gyroscope; picture-in-picture" allowfullscreen></iframe>
+<iframe width="560" height="315" src="https://www.youtube-nocookie.com/embed/8fi7uSYlOdc" frameborder="0" allow="accelerometer; autoplay; encrypted-media;gyroscope; picture-in-picture" allowfullscreen></iframe>
```

Comment:
Any reason this white space before gyroscope was removed?

Reply:
It was unintentional, I have updated it in the latest commit.